guardrails/schema.py

```diff
@@ -635,7 +635,7 @@ def validate(
         guard_logs.field_validation_logs = validation_logs
 
         validated_response, metadata = validator_service.validate(
-            value=data,
+            value=validated_response,
```

Comment:
Is there a reasonable/valuable way to test this; like an `assert called with` or something?  Or are we just testing that python does python stuff at that point?

Reply:
Just touched base with Nef about the current state of out test suite.  I think it would make the most sense to cover this in the unit tests for StringSchema/JsonSchema once we get to writing those.  Since we assert on the float values in the compiled prompt I think that's probably good enough for now.